packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less

```diff
@@ -7,21 +7,31 @@
   margin-top: 4px;
   position: relative;
 
-  .btn.btn-link {
+  a {
     color: @vertical-tab-pf-color;
+    display: inline-block;
     font-size: 13px;
     padding-left: 15px;
+    width: 100%;
+    word-break: break-word;
```

Comment:
This should be optional but enabled by default?

Reply:
As in a parameter to the tab for wrap or ellipse?

Reply:
Yeah.  Offer option so users can decided on an instance basis? ` none | break-word | truncate`